api/osrd_infra/migrations/0010_osrdtracksection.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.1.5 on 2023-02-20 17:26
+
+from django.db import migrations, models
+
+from osrd_infra.migrations import run_sql_create_infra_search_table
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("osrd_infra", "0009_tile_bbox"),
+    ]
+
+    operations = [
+        run_sql_create_infra_search_table(
+            name="osrd_search_tracksection",
+            source_table="osrd_infra_tracksectionmodel",
+            search_columns={
+                "line_name": "{source}.data#>>'{{extensions,sncf,line_name}}'",
+                "line_code": "{source}.data#>>'{{extensions,sncf,line_code}}'",
+            },
+            extra_columns={
```

Comment:
```suggestion
            },
            extra_columns={
                "line_code": "{source}.data#>>'{{extensions,sncf,line_code}}'",

```

makes `line_code` an integer